app/models/card.rb

```diff
@@ -44,6 +38,24 @@ def self.pending_cards_notification
     end
   end
 
+  def self.first_repeating_or_pending_card
+    card = pending.try(:first)
+    card ||= repeating.try(:first)
+    card
+  end
+
+  def handle_answer_distance(distance, sm_hash)
+    if distance <= 1
+      sm_hash.merge!({ review_date: Time.now + interval.to_i.days, attempt: 1 })
```

Comment:
Redundant curly braces around a hash parameter.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.